configs/testnet/core/dashd.conf

```diff
@@ -0,0 +1,30 @@
+# general
+
+testnet = 1
+
+daemon=0  # leave this set to 0 for Docker
+logtimestamps=1
+maxconnections=256
+
+# JSONRPC
+server=1
+rpcport=20002
+rpcuser=dashrpc
+rpcpassword=password
+
+rpcallowip=127.0.0.0
+rpcallowip=172.16.0.0/12
+rpcallowip=172.27.0.0/16
+rpcallowip=192.168.0.0/16
+
+rpcworkqueue=64
+rpcthreads=16
+
+# external network
+listen=1
+bind=0.0.0.0
+port=20001
```

Comment:
Let's remove it from dashd configs since we pass it as an option.

Reply:
Remove just the port, or all options relating to external network?